readme.md

```diff
@@ -1,692 +1,67 @@
-# Gataca-QR
+# Gataca-QR Components
```

Comment:
I think at the beginning of the readme it's necessary a big disclaimer notifying there are Breaking changes.

`
:warning: **Breaking Changes** [List of breaking changes from v1 to v2](#Breaking changes) :warning:
`